packages/phishing-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.1.0]
```

Comment:
@AugmentedMode Is there a reason why the minor part of the version was bumped here? It looks like this release only include a fix. Usually we use minor to communicate new functionality that's introduced, but for fixes we bump the patch part of the version.

Reply:
Hey I agree that this release should of been a patch, I will take this going forward. Thanks @mcmire